caddyhttp/proxy/upstream.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	supportedPolicies = make(map[string]func() Policy)
+	supportedPolicies = make(map[string]func(string) Policy)
```

Comment:
Or maybe `func(args ...string) Policy` to make it dynamic in case of any future policy that may require more than one args.